app/src/main/java/app/pachli/settings/SettingsConstants.kt

```diff
@@ -103,6 +103,7 @@ object PrefKeys {
 
     const val TAB_FILTER_HOME_REPLIES = "tabFilterHomeReplies_v2" // This was changed once to reset an unintentionally set default.
     const val TAB_FILTER_HOME_BOOSTS = "tabFilterHomeBoosts"
+    const val TAB_FILTER_HOME_SELF_BOOSTS = "tabFilterHomeSelfBoosts"
```

Comment:
```suggestion
    const val TAB_SHOW_HOME_SELF_BOOSTS = "tabShowHomeSelfBoosts"
```

Per earlier comment.

Reply:
Per earlier suggestion.